example-get-started-experiments/code/.github/workflows/run-studio-experiment.yml

```diff
@@ -38,20 +42,17 @@ jobs:
       options: --gpus all --ipc host
     steps:
       - uses: actions/checkout@v3
-      - uses: aws-actions/configure-aws-credentials@v1
-        with:
-          aws-region: us-east-1
-          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
       - name: training
         env:
           REPO_TOKEN: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
           STUDIO_TOKEN: ${{ secrets.STUDIO_TOKEN }}
           DVCLIVE_LOGLEVEL: DEBUG
         run: |
           pip install -r requirements.txt
-          dvc pull
+          dvc pull -R data 
+          cml ci --fetch-depth 0
           cml comment create --publish --watch results/train/report.md & dvc exp run
-          dvc push
+          dvc remote add --local push_remote s3://dvc-public/remote/get-started-pools 
+          dvc push -r push_remote
```

Comment:
Still trying to figure out how to make this push work with the sandbox role.

Reply:
Will the `example-get-started-experiments` repo have sufficient permissions to write to `s3://dvc-public/remote/get-started-pools `?